brave/src/test/java/brave/features/handler/CountingChildrenTest.java

```diff
@@ -136,12 +135,12 @@ public boolean begin(TraceContext context, MutableSpan span, @Nullable TraceCont
     assertThat(nameToChildCount)
       .isEqualTo(nameToChildCountDirect)
       .containsExactly(
-        tuple("root1child1child2child1", "0"),
-        tuple("root2child1", "0"),
-        tuple("root1child1child1", "1"),
+        tuple("root1Child1Child2Child1", "0"),
```

Comment:
a lot of tests will drift like this, but it is good as it proves brave doesn't meddle with case format even though our zipkin handler does.